README.md

```diff
@@ -216,6 +216,20 @@ The execution of integration tests require the following environment variables s
 * For default account systems (OAuth): `CHECKOUT_DEFAULT_OAUTH_CLIENT_ID` & `CHECKOUT_DEFAULT_OAUTH_CLIENT_SECRET`
 * For Previous account systems (ABC): `CHECKOUT_PREVIOUS_PUBLIC_KEY` & `CHECKOUT_PREVIOUS_SECRET_KEY`
 
+## Telemetry
+Request telementry is enabled by default in the Go SDK. Request latency is included in the telemetry data. Recording the request latency allows Checkout.com to continuously monitor and imporove the merchant experience.
```

Comment:
Should we add a note saying the information is sent back in the header?  That would make it clear that the SDK is not using any additional resources (e.g. making another HTTP call) to pass back the telemetry data